### PR TITLE
implementation of Movie, MovieSession, and Authentication controllers

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

File: src/main/java/cinema/config/AppConfig.java [hunk 1]
```diff
@@ -15,7 +15,8 @@
 @ComponentScan(basePackages = {
         "cinema.service.impl",
         "cinema.dao.impl",
-        "cinema.security"
+        "cinema.security",
+        "cinema.model.dto"
```
Comment (dariiabutorina) [inline src/main/java/cinema/config/AppConfig.java:19]: Does it work without this line?

File: src/main/java/cinema/dao/impl/MovieDaoImpl.java [hunk 2]
```diff
@@ -52,4 +52,13 @@ public Movie add(Movie movie) {
...
         }
     }
+
+    @Override
+    public Movie get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            return session.get(Movie.class, id);
+        } catch (Exception e) {
+            throw new DataProcessingException("Couldn't get movie with id " + id, e);
+        }
```
Comment (dariiabutorina) [inline src/main/java/cinema/dao/impl/MovieDaoImpl.java]: Do we need the catch block on the READ operations?

File: src/main/java/cinema/controller/MovieSessionController.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    @PostMapping
+    public MovieSession addMovieSession(
+            @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        return movieSessionService.add(movieSessionDtoMapper
+                .mapToMovieSession(movieSessionRequestDto));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getAllAvailableSessions(Long movieId, String date) {
```
Comment (RomanDubovskyi) [inline src/main/java/cinema/controller/MovieSessionController.java]: this is optional, but you can consider following approach 
@RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date

File: src/main/java/cinema/model/dto/MovieResponseDto.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
+package cinema.model.dto;
+
+public class MovieResponseDto {
+    private Long id;
+    private String title;
+
```
Comment (RomanDubovskyi) [inline src/main/java/cinema/model/dto/MovieResponseDto.java:7]: what about description?

**Review by RomanDubovskyi — COMMENTED** [review]
all good, just add description to movieResponseDto

File: src/main/java/cinema/controller/MovieSessionController.java (hunk 3 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/controller/MovieSessionController.java]: done

File: src/main/java/cinema/model/dto/MovieResponseDto.java (hunk 4 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/model/dto/MovieResponseDto.java:7]: done

File: src/main/java/cinema/config/AppConfig.java (hunk 1 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/config/AppConfig.java:19]: no

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**